src/main_app/components/Logo.js

```diff
@@ -1,40 +1,20 @@
-function Logo ({ className, variant = 'full' }) {
-  const showText = variant === 'full'
+import useVariants, { combineVariants, isVariant } from 'utils/use_variants'
+import SVGIcon from 'main_app/components/SVGIcon'
 
-  let viewBox
-  let width
-  let height
-  if (showText) {
-    viewBox = '0 0 196 42'
-    width = '196'
-    height = '42'
-  } else {
-    viewBox = '0 0 50 42'
-    width = '50'
-    height = '42'
+function Logo ({ className, logoVariant = 'full', variant }) {
```

Comment:
Usa sólo una variante por componente. `logoVariant` no se está usando ahora, sólo deja `variant`.
```suggestion
function Logo ({ className, variant }) {
```